Makefile.am

```diff
@@ -1,2 +1,11 @@
+# This file is part of lustre-tools-llnl.
+#
+# SPDX-License-Identifier: GPL-2.0
+# See [LICENSE-GPL-2.0](https://github.com/LLNL/lustre-tools-llnl/LICENSE-GPL-2.0)
+#
+# Copyright (c) 2011, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+# LLNL-CODE-468512
+
 SUBDIRS = src scripts doc
 EXTRA_DIST = COPYING lustre-tools-llnl.spec
```

Comment:
This mentions 'COPYING' but did we just rename that file to LICENSE-GPL-2.0?

Reply:
Good catch!